examples/webgpu_lights_custom.html

```diff
@@ -113,6 +113,7 @@
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setAnimationLoop( animate );
+				renderer.toneMapping = THREE.LinearToneMapping;
```

Comment:
When an application produces colors in the fragment shader that exceed the `[0,1]` value range (so values in the open domain range `[0, ∞]`), it must use tone mapping otherwise the sRGB conversion is incorrect.